src/main/java/mesander/com/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,54 @@
+package mesander.com.TechItEasy.controllers;
+
+import mesander.com.TechItEasy.Television;
+import mesander.com.TechItEasy.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    private ArrayList<Television> televisions = new ArrayList<>();
+
+    @GetMapping("/televisions")
+    public ResponseEntity<ArrayList<Television>> getAllTelevisions() {
+        return new ResponseEntity<>(this.televisions, HttpStatus.OK);
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Television> getTelevision(@PathVariable int id) {
+        if (id >= 0 && id < this.televisions.size()) {
+            return new ResponseEntity<>(this.televisions.get(id), HttpStatus.OK);
+        } else {
+            throw new RecordNotFoundException("ID cannot be found");
+        }
+    }
+
+    @PostMapping("/televisions")
+    public ResponseEntity<Television> addTelevision(@RequestBody Television television) {
```

Comment:
Je had hier ook automatisch het id kunnen zetten door de size van je arraylist op te halen (Later gaan we dit genereren)